bower.json

```diff
@@ -21,5 +21,6 @@
 		, "fontawesome": "4.3.0"
 		, "knockout": "3.3.0"
 		, "bootbox": "4.4.0"
+		, "chosen": "harvesthq/chosen#~1.4.2"		
```

Comment:
add ```chosen``` as dependency into bower file so we do not need to make separate call for installation